etna/packages/etna-js/utils/asserts.jsx

```diff
@@ -2,12 +2,18 @@ export function assertIsSome(thing) {
   const keys = Object.keys(thing);
 
   if (keys.length === 1) {
-    const key = keys[0];
-    const value = thing[key];
+    const value = thing[keys[0]];
     if (!value) {
       throw new Error(`Expected ${key} to be present, value was ${value}`);
     }
     return value;
+  } else if (keys.length > 1) {
```

Comment:
Convenience method I'm writing to help hunting down stray null bugs faster.

assertIsSome({ my, inputs, here }) => { my, inputs, here } and verify each key is true-y.